syncopy/io/load_tdt.py

```diff
@@ -0,0 +1,700 @@
+                        #         elif isinstance(store, list):
+                        #             if buddy in store:
+                        #                 skip_by_name = False
+                        # if skip_by_name:
+                        #     continue
+                        epocs.name.append(store_code['name'])
+                        epocs.buddies.append(buddy)
+                        epocs.code.append(store_code['code'])
+                        epocs.ts.append([])
+                        epocs.type.append(store_code['epoc_type'])
+                        epocs.type_str.append(store_code['type_str'])
+                        epocs.data.append([])
+                        epocs.dform.append(store_code['dform'])
+
+                if not var_name in header.stores.keys():
+                    if store_code['type_str'] != 'epocs':
+                        header.stores[var_name] = StructDict(name = store_code['name'],
+                                                             code = store_code['code'],
+                                                             size = store_code['size'],
+                                                             type = store_code['type'],
+                                                             type_str = store_code['type_str'])
+                        if header.stores[var_name].type_str == 'streams':
+                            header.stores[var_name].ucf = store_code['ucf']
+                        if header.stores[var_name].type_str != 'scalars':
+                            # Finding the sampling rate
+                            header.stores[var_name].fs = np.double(np.array([store_code['temp'][9]]).view(np.float32))
+                        header.stores[var_name].dform = store_code['dform']
+                valid_ind = np.where(codes == store_code['code'])[0]
+                temp = heads[3, valid_ind].view(np.uint16)
+
+                if store_code['type_str'] != 'epocs':
+                    if not hasattr(header.stores[var_name], 'ts'):
+                        header.stores[var_name].ts = []
+                    vvv = np.reshape(heads[[[4], [5]], valid_ind].T, (-1, 1)).T.view(np.float64) - header.start_time
+                    # round timestamps to the nearest sample
+                    vvv = self.time2sample(vvv, to_time = True)
+                    header.stores[var_name].ts.append(vvv)
+                    if (not self.nodata) or (store_code['type_str'] == 'streams'):
+                        if not hasattr(header.stores[var_name], 'data'):
+                            header.stores[var_name].data = []
+                        header.stores[var_name].data.append(np.reshape(heads[[[6], [7]], valid_ind].T, (-1, 1)).T.view(np.float64))
+                    if not hasattr(header.stores[var_name], 'chan'):
+                        header.stores[var_name].chan = []
+                    header.stores[var_name].chan.append(temp[::2])
+                else:
```

Comment:
ah here is an `else`.. mmh, I guess the logical program flow could be optimized here